homeassistant/components/calendar/services.yaml

```diff
@@ -52,3 +52,22 @@ create_event:
       example: "Conference Room - F123, Bldg. 002"
       selector:
         text:
+list_events:
+  name: List event
+  description: List events on a calendar within a time range.
+  target:
+    entity:
+      domain: calendar
+  fields:
+    start_date_time:
```

Comment:
I bet that the main use case for this function is going to be to query for next X hours. Wouldn’t it make more sense to take in a duration as a field ?

Reply:
Looks good, I think.